### PR TITLE
Add SLERT product entry to Installer (jsc#PM-3549, jsc#SLE-24980)

### PR DESCRIPTION
**Pull request by JeffreyCheung**

### CODE REVIEW

File: control/control.leanos.xml [hunk 1]
```diff
@@ -176,6 +176,13 @@ textdomain="control"
             <version>15.4</version>
             <register_target>sle-15-$arch</register_target>
           </base_product>
+          <base_product>
+  	    <display_name>SUSE Linux Enterprise Real Time 15 SP4</display_name>
+	    <name>SLE_RT</name>
+	    <version>15.4</version>
+	    <register_target>sle-15-$arch</register_target>
+	    <archs>x86_64</archs>
+ 	  </base_product>
```
Comment (dgdavid) [inline control/control.leanos.xml]: Let's keep format respecting the expected indentation

```suggestion
  	        <display_name>SUSE Linux Enterprise Real Time 15 SP4</display_name>
	        <name>SLE_RT</name>
	        <version>15.4</version>
	        <register_target>sle-15-$arch</register_target>
	        <archs>x86_64</archs>
 	    </base_product>
```

**Comment by dgdavid** [comment]
Hi!

Apart from right indentation, I miss the version bump and the entry in the changelog.  Let me know if you need help with that.

Thanks!

**Comment by JeffreyCheung** [comment]
> Hi!
> 
> Apart from right indentation, I miss the version bump and the entry in the changelog. Let me know if you need help with that.
> 
> Thanks!

Hi David, yes please do help.

**Review by ancorgs — APPROVED** [review]
LGTM

**Pull request merged**